team-frontend/src/components/QuizMasterGame.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div id="content">
+    <h1>Welkom in de Game omgeving, {{userName}}!</h1>
+    <b-container fluid>
+      <b-col>{{game.gameTitle}} ({{game.state}})</b-col>
+    </b-container>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+import { Route } from "vue-router";
+import store from "../store";
+import { AxiosResponse, AxiosError } from "axios";
+import { ApiResponse } from "../models/models";
+
+@Component({
+  beforeRouteEnter(to: Route, from: Route, next: any) {
+    // called before the route that renders this component is confirmed.
+    // does NOT have access to `this` component instance,
+    // because it has not been created yet when this guard is called!
+
+    if (!store.state.isLoggedIn) {
+      next("/");
+    }
+    // todo also check the state of the game, you might want to go straight back into the game.
+    next();
+  }
+})
+
+export default class QuizMasterGame extends Vue {
+  public name: string = "QuizMasterGame";
+
+  public created() {
+    // // get QuizMaster Game view model
+    // this.$axios
+    //   .get("/api/game/quizmastergame")
+    //   .then((response: AxiosResponse<QuizMasterGameViewModel>) => {
+    //   })
+    //   .catch((error: AxiosError) => {
+    //     this.$bvToast.toast(error.message, {
+    //       title: "oops",
+    //       variant: "error"
+    //     });
+    //   });
+  }
```

Comment:
Als dead code niet meer gebruikt gaat worden kan het weg.

Reply:
Geen deadcode, maar copy/pasted en nog niet nodig. Maakt het niet veel beter, maar wordt nog opgeruimd. Erger is, dat dit al een volgend issue is die ik al gecommit heb zonder eerste een nieuwe branch te maken of pull request af te wachten. Beetje zootje aan het maken van Git.  WIP in mijn hoofd.